ui/apps/platform/src/Containers/NetworkGraph/TopologyComponent.tsx

```diff
@@ -170,13 +175,10 @@ const TopologyComponent = ({
                 <TopologySideBar resizable onClose={closeSidebar}>
                     {simulation.isOn && simulation.type === 'networkPolicy' && (
                         <NetworkPolicySimulatorSidePanel
-                            networkPolicyGenerationScope={getInScopeEntities(
-                                model.nodes,
-                                scopeHierarchy
-                            )}
                             simulator={simulator}
                             setNetworkPolicyModification={setNetworkPolicyModification}
                             scopeHierarchy={scopeHierarchy}
+                            scopeDeploymentCount={deploymentCount ?? 0}
```

Comment:
Off the subject of this change, do you know why 2 conditional renders at such different levels?
* `NetworkGraphPage` renders `NetworkPolicySimulatorSidePanel` directly
* `NetworkGraphPage` renders `NetworkGraphContainer` which renders `NetworkGraph` which renders `TopologyComponent` which renders `NetworkPolicySimulatorSidePanel` here

Reply:
The second case is the original usage of the `NetworkPolicuSimulatorSidePanel`, but with the recent set of changes it was also required to render apart from the main visualization. (We need the ability to simulate network policies at the cluster level, even if we are not rendering nodes on the graph.)

It looks relatively straightforward to refactor this though so that we have a single rendering at the top level instead of multiple usages. I'll add this to my TODO list.